lints/lint_rsa_mod_factors_smaller_than_752.go

```diff
@@ -26,7 +26,11 @@ func (l *rsaModSmallFactor) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *rsaModSmallFactor) RunTest(c *x509.Certificate) (ResultStruct, error) {
-	mod := c.PublicKey.(*rsa.PublicKey).N
```

Comment:
This file name should probably be `lints/lint_rsa_mod_factors_smaller_than_752_bits.go` to match how all the other files are named.